library/maven/artifacts.bzl

```diff
@@ -234,30 +234,31 @@ artifacts = {
     "org.hamcrest:hamcrest-core": "1.3",
     "org.hamcrest:hamcrest-library": "1.3",
     "org.javatuples:javatuples": "1.2",
-    "org.jetbrains.compose.animation:animation-core-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.desktop:desktop-jvm": "1.1.0-alpha04",
-    "org.jetbrains.compose.foundation:foundation-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.foundation:foundation-layout-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.material:material-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.ui:ui-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.ui:ui-geometry-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.ui:ui-graphics-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.ui:ui-test-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.ui:ui-test-junit4-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.ui:ui-text-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.ui:ui-unit-desktop": "1.1.0-alpha04",
-    "org.jetbrains.compose.runtime:runtime-desktop": "1.1.0-alpha04",
-    "org.jetbrains.kotlin:kotlin-reflect": "1.6.0",
-    "org.jetbrains.kotlin:kotlin-stdlib-jdk7": "1.6.0",
-    "org.jetbrains.kotlin:kotlin-test": "1.6.0",
-    "org.jetbrains.kotlinx:kotlinx-coroutines-core": "1.6.0",
-    "org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm": "1.6.0",
-    "org.jetbrains.kotlinx:kotlinx-coroutines-test": "1.6.0",
+    "org.jetbrains.compose.compiler:compiler": "1.3.2",
```

Comment:
Compose compiler as a maven import! Horray!

Reply:
Great! Can we get a usage example?

Reply:
No longer applicable in the same way, but I'll explain the new usage:
* We specify the compiler plugin as a dependency as above.
* We specify the compiler plugin as a `kt_compiler_plugin` for consumption by dependors in '//builder/compose/BUILD`.
* We make use of this plugin via methods such as `kt_jvm_library` adding the argument `plugins = ["@vaticle_dependencies//builder/compose:compiler_plugin"]`.

Reply:
Let's copy in the relevant code snippets (that we already wrote in the Studio PR) for reference to future readers here

Reply:
Done.